src/Sylius/Behat/Service/Setter/CookieSetter.php

```diff
@@ -31,7 +32,7 @@ public function setCookie($name, $value)
     {
         $driver = $this->minkSession->getDriver();
 
-        if ($driver instanceof ChromeDriver) {
+        if ($driver instanceof ChromeDriver || $driver instanceof PantherDriver) {
```

Comment:
About the idea of a new service, a specific method allowing to test for auto-start support can be implemented too here.

Reply:
We can consider it and push in a separate PR. It isn't too much related to this one :P.